thesis-fe/src/components/container/CalendarDashboard.js

```diff
@@ -102,16 +108,20 @@ class MyTripsDashboard extends Component {
     const toShow = this.state.listInfo.length > 0 && (this.state.selectedUser)
     return (
       <Container>
+        <BIG>
+          Calendar
+        </BIG>
         <ContainerUsers>
         {this.state.listInfo}
         </ContainerUsers>
         <H2>
           Checking {this.state.selectedUser}'s avaiability
         </H2>
         <DateRange
-          startDate={(toShow && mock[toShow][0]) || null}
-          endDate={(toShow && mock[toShow][1]) || null}
-          onInit={null}
+          minDate={(toShow && moment(this.state.usersInfo[toShow].startDate).format('DD-MM-YYYY')) || null}
```

Comment:
I had to change the format to YYYY-MM-DD  in the wizard to match de backend mutations . I think we should follow  the same format in the different pages.